### PR TITLE
fix entity that extends external entity

### PR DESCRIPTION
**Pull request by larisaVasile**

### CODE REVIEW

**Review by michelu89 — CHANGES_REQUESTED** [review]

File: core/libs/instantiator/src/lib/instantiators/entity-instantiator.ts [hunk 1]
```diff
@@ -54,6 +54,7 @@ export class EntityInstantiator {
 
         if (externalElement) {
           defaultEntity.extendedElement = externalElement;
+          defaultEntity.children.push(externalElement);
```
[the remark this answers is not in the document]
Comment (larisaVasile) [reply core/libs/instantiator/src/lib/instantiators/entity-instantiator.ts:57]: done

File: core/apps/ame-e2e/src/fixtures/external-reference/same-namespace/model-with-extended-entity.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:org.eclipse.examples:1.0.0#> .
+
+:SimpleAspect a samm:Aspect ;
```
Comment (michelu89) [inline core/apps/ame-e2e/src/fixtures/external-reference/same-namespace/model-with-extended-entity.txt:10]: Use here also the same naming "AspectWithExtendedEntity" or something

File: core/apps/ame-e2e/src/fixtures/external-reference/same-namespace/model-with-extended-entity.txt [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+   samm:properties ( :simpleProperty1 ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:simpleProperty1 a samm:Property ;
```
Comment (michelu89) [inline core/apps/ame-e2e/src/fixtures/external-reference/same-namespace/model-with-extended-entity.txt:15]: Remove also simple, in each naming.

File: core/apps/ame-e2e/src/integration/editor/entity-that-extends-external-entity.cy.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+/// <reference types="Cypress" />
+
+import {SELECTOR_notificationsBtn, SELECTOR_notificationsDialogCloseButton} from '../../support/constants';
+
+describe('Test loading aspect with extended external Entity', () => {
```
Comment (michelu89) [inline core/apps/ame-e2e/src/integration/editor/entity-that-extends-external-entity.cy.ts:18]: Include also same tests for different namespaces and maybe also for cross namespaces.

**Review by michelu89 — CHANGES_REQUESTED** [review]

File: core/apps/ame-e2e/src/fixtures/external-reference/same-namespace/model-with-extended-entity.txt (hunk 2 above)
Comment (larisaVasile) [reply core/apps/ame-e2e/src/fixtures/external-reference/same-namespace/model-with-extended-entity.txt:10]: done

File: core/apps/ame-e2e/src/fixtures/external-reference/same-namespace/model-with-extended-entity.txt (hunk 3 above)
Comment (larisaVasile) [reply core/apps/ame-e2e/src/fixtures/external-reference/same-namespace/model-with-extended-entity.txt]: done

File: core/apps/ame-e2e/src/integration/editor/entity-that-extends-external-entity.cy.ts (hunk 4 above)
Comment (larisaVasile) [reply core/apps/ame-e2e/src/integration/editor/entity-that-extends-external-entity.cy.ts:18]: done

**Pull request merged**